general/1_terminal/anatolii-kravchenko/task-3.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# This scrpt is intended for print today's day of the month (as number), 
+# and to comment: is it working day or weekend.
+
+dayNumber=$(date +"%d")                                     # Mashine number of the day
+printDayNumber=${dayNumber#"${dayNumber%%[!0]*}"}           # Removed leading zero from Mashine number
+
+case $(date +"%u") in
+    1) echo "Looks like ${printDayNumber} is a working day" # Monday
+        ;;
+    2) echo "Looks like ${printDayNumber} is a working day" # Tuesday
+        ;;
+    3) echo "Looks like ${printDayNumber} is a working day" # Wednesday
+        ;;
+    4) echo "Looks like ${printDayNumber} is a working day" # Thursday
+        ;;
+    5) echo "Looks like ${printDayNumber} is a working day" # Friday
+        ;;
+    6) echo "Looks like ${printDayNumber} is a weekend"     # Saturday
```

Comment:
Nice of you to find the `case` statement that was not in the lecture.

However, I think we can find a better approach here. Look, for example, at how we repeat the same string 5 times. 
What you wrote here, essentially, is "for numbers between 1 and 5 do _this_, and for numbers 6 to 7 do _that_". We don't necessarily need to enumerate every number.

Reply:
I changed it to the IF statement, in last commit.